volatility/framework/symbols/windows/extensions/__init__.py

```diff
@@ -790,12 +793,29 @@ def get_sids(self) -> Iterable[str]:
 
 
     def privileges(self):
-        "Return a list of privileges for the current token object."
-        for priv_index in range(64):
-            yield (priv_index,
-                   bool(self.Privileges.Present & (2**priv_index)),
-                   bool(self.Privileges.Enabled & (2**priv_index)),
-                   bool(self.Privileges.EnabledByDefault & (2**priv_index)))
+        """Return a list of privileges for the current token object."""
+        
+        try:
+            for priv_index in range(64):
+                yield (priv_index,
+                       bool(self.Privileges.Present & (2**priv_index)),
+                       bool(self.Privileges.Enabled & (2**priv_index)),
+                       bool(self.Privileges.EnabledByDefault & (2**priv_index)))
+        except AttributeError: # Windows XP
+            layer_name = self.vol.layer_name
+            kvo = self._context.layers[layer_name].config["kernel_virtual_offset"]        
+            symbol_table = self.get_symbol_table_name()
+            ntkrnlmp = self._context.module(symbol_table,
```

Comment:
I don't think you need to make a module, which also means you don't need to load the kvo.  I think you should just be able to ask for `self.context.symbol_space[self.get_symbol_table_name()].get_type("_LUID_AND_ATTRIBUTES")` to get the subtype.  That should remove a lot of these extra lines. and hopefully speed up the call...